Fundamentals/Setup/Upgrading/umbraco10-breaking-changes.md

```diff
@@ -0,0 +1,149 @@
+
+## v10 scope provider cleanup
+
+- Some public classes in the _Cms.Core.Services_ namespace have moved assembly from **_Umbraco.Cms.Infrastructure_** to **_Umbraco.Cms.Core_**
+
+- These same public classes have changed namespace from **_Umbraco.Cms.Core.Services.Implement_** to **_Umbraco.Cms.Core.Services_**.
+
+## v10 update to npoco5
+
+NPoco types and interfaces are part of our public interface and there are changes upstream.
+
+## v10 SQLite support + distributed locking abstractions
+
+- Removed support for Microsoft SQL Server Compact (SQL CE)
+
+- Removed ReadLock & WriteLock methods from _ISqlSyntaxProvider_ interface, please instead make use of - _IDistributedLockingMechanism_ (or IScope which delegates to _IDistributedLockingMechanism_).
+
+- Constants for SQL Server provider name moved+consolidated from _Core.Constants.DatabaseProviders_ & _Core.Constants.-DbProviderNames_ to _Umbraco.Cms.Persistence.SqlServer.Constants_
+
+- Various SQL Server related services moved from the Umbraco.Infrastructure project to the new Umbraco.Cms.
+Persistence.- SqlServer project with altered namespaces e.g.
+
+- _SqlServerSyntaxProvider_, _SqlServerBulkSqlInsertProvider_, _SqlServerDatabaseCreator_
+
+**Added various methods / properties to ISqlSyntaxProvider, these must be implemented in any downstream implementation e.g:**
+
+- _ISqlSyntaxProvider.HandleCreateTable(IDatabase,TableDefinition,Boolean)_
+- _ISqlSyntaxProvider.GetFieldNameForUpdate()_
+- _ISqlSyntaxProvider.GetColumn(DatabaseType,String,String,String,String,Boolean)_
+- _ISqlSyntaxProvider.InsertForUpdateHint(Sql)_
+- _ISqlSyntaxProvider.AppendForUpdateHint(Sql)_
+- _ISqlSyntaxProvider.LeftJoinWithNestedJoin(Sql,Func<Sql,Sql>,String)_
+
+## v10: Update to ImageSharp v2
+
+**Update dependency versions**:
+
+- _SixLabors.ImageSharp from 1.0.4 to 2.1.1 (see breaking changes in v2.0.0 release notes);_
+
+- _SixLabors.ImageSharp.Web from 1.0.5 to 2.0.0 (see breaking changed in release notes);_
+
+Renamed CachedNameLength property to CacheHashLength on ImagingCacheSettings.
+
+Moved ImageSharpImageUrlGenerator from project Umbraco.Infrastructure to Umbraco.Web.Common and updated the corresponding namespace and DI registration (from AddCoreInitialServices() to AddUmbracoImageSharp());
+
+Moved ImageSharp configuration from the _AddUmbracoImageSharp()_ extension method into separate _IConfigureOptions<>_ implementations:
+
+- _The middleware is configured in ConfigureImageSharpMiddlewareOptions (which also replaces ImageSharpConfigurationOptions that previously only set the default ImageSharp configuration);_
+
+- _The default physical cache is configured in ConfigurePhysicalFileSystemCacheOptions._
+
+## V10: Migrate member properties to columns on the member table
+
+This is breaking because it is no longer possible access the properties listed below through the _IMember.Properties_ collection, you must now access them through their specific properties I.E. _IMember.IsLockedOut_.
```

Comment:
```suggestion
This is breaking because it is no longer possible to access the properties listed below through the _IMember.Properties_ collection. You must now access them through their specific properties i.e. _IMember.IsLockedOut_.
```